node/src/test_utils/mod.rs

```diff
@@ -490,6 +494,20 @@ pub fn make_wallet(address: &str) -> Wallet {
     Wallet::from_str(&dummy_address_to_hex(address)).unwrap()
 }
 
+pub fn make_daemon_bind_message(ui_gateway: Recorder) -> DaemonBindMessage {
```

Comment:
Please move `make_daemon_bind_message` inside module `unshared_test_utils` which is in the same file.